tests/manager.rs

```diff
@@ -378,3 +378,97 @@ fn test_safe_mode_corrupt_while_open_2() {
         Some(Value::Str("byé, yöu"))
     );
 }
+
+/// Test how the manager can discard corrupted databases, while moving the corrupted one aside for
+/// later inspection.
+#[test]
+fn test_safe_mode_corrupt_while_open_3() {
+    type Manager = rkv::Manager<SafeModeEnvironment>;
+
+    let root = Builder::new()
+        .prefix("test_safe_mode_corrupt_while_open_3")
+        .tempdir()
+        .expect("tempdir");
+    fs::create_dir_all(root.path()).expect("dir created");
+
+    // Verify it was flushed to disk.
```

Comment:
I see you copypasted this line from `test_safe_mode_corrupt_while_open_1`, but this comment doesn't seem to make sense when the `exists()` call now happens after the write call.